src/components/dialog/Dialog.tsx

```diff
@@ -4,11 +4,15 @@ import omit from 'lodash.omit';
 import React, { ReactNode, useCallback, useEffect, useRef, useState } from 'react';
 import { useResizeDetector } from 'react-resize-detector';
 import { GenericComponent } from '../../@types/types';
-import { Button, ButtonGroup, DialogBase, Heading3 } from '../../index';
+import { Button, ButtonGroup, ButtonProps, DialogBase, Heading3, Toggle, ToggleProps } from '../../index';
 import { DialogBaseProps } from './DialogBase';
 import theme from './theme.css';
 import { SIZES } from '../../constants';
 
+interface LeftAction extends Omit<ButtonProps | ToggleProps, 'fullWidth' | 'marginTop'> {
+  isToggle?: boolean;
+}
```

Comment:
PP: feels a bit weird to have the `isToggle` together with the button/toggle props.